### DISCUSSION TITLE
How do i disable the prefix `eslint-config-` when importing local eslint config package

### DISCUSSION THREAD

**Discussion by vexCoder** [eslint/eslint#15828] [Q&A] [answered]
I wanted to merge all the config files like `tsconfig.json` and `.eslintrc` in a separate yarn workpace package.
for example i have a package name `config`. i wanna be able to extend `config/react` or `config/node` in another package `.eslintrc` file while im still able to extend in `tsconfig.json` like `config/tsconfig.base.json`.

while this is fairly easy to achieve and possible, in order for me to be able to extend in an eslint config file i need to prefix the package name with `eslint-config-`. so the package name needs to be `eslint-config-config`.

this pose a problem for me since in the tsconfig i have to write `eslint-config-config/tsconfig.base.json`

and calling a package name `eslint-config-config` while distributing tsconfig files feels a little bit wrong.

is there a workaround to this so that i dont have to prefix it with `eslint-config-`?

**Comment by nzakas** [comment, marked as answer]
There isn’t currently a workaround. You can optionally create a plugin with the configs you want to share, so you could have eslint-plugin-config and then use “plugin:config/node” in your extends field.

**Reply by vexCoder** [reply]
Thank you for answering not really what i need but i learned eslint plugins because of this haha.
i guess ill have to separate tsconfig and eslint configs.

**Comment by cjayyy** [comment]
It looks like this problem disappears with new flat configurations - see https://eslint.org/docs/latest/use/configure/configuration-files-new.
